### DISCUSSION TITLE
能否带参启动cmd

### DISCUSSION THREAD

**Discussion by OrdinaryChen** [RipplePiam/MobaXterm-Chinese-Simplified#55] [Q&A]
最近做esp32的代码 他的idf工具是使用C:\Windows\system32\cmd.exe /k ""D:\Espressif\idf_cmd_init.bat" esp-idf-7e66731f0c5ae58ca35bfad7cd15a1a5" 参数启动的cmd  而不是进入一个cmd输入这些参数。mobaxterm在哪里设置可以带参数的打开一个cmd啊

**Comment by RipplePiam** [comment]
新版本终端会话设置里都有一个额外参数可以满足需求
